src/status_im/signals/core.cljs

```diff
@@ -53,10 +55,27 @@
                 :peer-stats  peer-stats
                 :peers-count (count (:peers peer-stats)))}))
 
-(defn handle-local-pairing-signals
-  [event]
-  (log/debug "local pairing signal received"
-             {:event event}))
+(rf/defn handle-local-pairing-signals
+  [{:keys [db] :as cofx} event]
+  (log/info "local pairing signal received"
+            {:event event})
+  (let [connection-success?          (= (:type event) constants/local-pairing-event-connection-success)
+        completed-pairing?           (and (= (:type event) constants/local-pairing-event-process-success)
+                                          (= (:action event)
+                                             constants/local-pairing-action-pairing-account))
+        logged-in?                   (multiaccounts.model/logged-in? cofx)
```

Comment:
I agree that the function doesn't need the entire `cofx`. Since this is an existing function, I had no other choice rather send the `cofx`.

I will create a separate PR to clean this function and usage across the codebase. :smiley: